linux_kinect/Bit data
---
sensor_msgs/PointCloud2 points